internal/rbd/replicationcontrollerserver_test.go

```diff
@@ -432,3 +434,34 @@ func TestCheckRemoteSiteStatus(t *testing.T) {
 		})
 	}
 }
+
+func TestValidateLastSyncTime(t *testing.T) {
+	t.Parallel()
+	tests := []struct {
+		name        string
+		description string
+		timestamp   *timestamppb.Timestamp
+	}{
+		{
+			"valid description",
+			//nolint:lll // sample output cannot be split into multiple lines.
+			`replaying,{"bytes_per_second":0.0,"bytes_per_snapshot":149504.0,"local_snapshot_timestamp":1662655501,"remote_snapshot_timestamp":1662655501}`,
+			timestamppb.New(time.Unix(1662655501, 0)),
+		},
+		{
+			"invalid description",
+			`replaying,{"bytes_per_second":0.0,"bytes_per_snapshot":149504.0,"remote_snapshot_timestamp":1662655501}`,
+			nil,
+		},
```

Comment:
please also add a test-case where `description` is an empty string (`""`), and contains some invalid JSON  (like missing `}`).